javascript/webpacker_react-npm-module/src/index.js

```diff
@@ -119,16 +120,15 @@ var WebpackerReact = {
     this.wrapForHMR = wrapForHMR
   },
 
-  register: function (component) {
-    var name = component.name
-
-    if (this.registeredComponents[name]) {
-      console.warn('webpacker-react: Cant register component, another one with this name is already registered: ' + name)
-      return false
+  register: function (components) {
+    var collisions = _.intersection(_.keys(this.registeredComponents), _.keys(components));
```

Comment:
Should we really check against collisions? I think it's overprotective and usually there will be one place in the app for registering components.